crates/turborepo-lib/src/run/scope/change_detector.rs

```diff
@@ -109,8 +109,47 @@ impl<'a> GitChangeDetector for ScopeChangeDetector<'a> {
                         (
                             name.to_owned(),
                             PackageInclusionReason::All(AllPackageChangeReason::GitRefNotFound {
-                                from_ref: from_ref.clone(),
-                                to_ref: to_ref.clone(),
+                                from_ref,
+                                to_ref,
+                            }),
+                        )
+                    })
+                    .collect());
+            }
+            Err(ScmError::Path(err, _)) => {
+                warn!(
+                    "Could not process some file paths: {}. Defaulting to all packages changed.",
+                    err
+                );
+                debug!("path error: {}, defaulting to all packages changed", err);
+                return Ok(self
+                    .pkg_graph
+                    .packages()
+                    .map(|(name, _)| {
+                        (
+                            name.to_owned(),
+                            PackageInclusionReason::All(AllPackageChangeReason::GitRefNotFound {
```

Comment:
Can you change this variant now that it is for many different `git` related failures instead of just the ref not being found?